File: CS/WinWebSolution.Module.Win/Child.cs
using DevExpress.Persistent.BaseImpl;
using DevExpress.Xpo;
using System;

namespace WinWebSolution.Module.Win {
    public class Child : BaseObject {
        public Child(Session session) : base(session) { }
        private string _name;
        public string Name {
            get { return _name; }
            set { SetPropertyValue("Name", ref _name, value); }
        }
        private Master _master;
        [Association("Master-Children")]
        public Master Master {
            get { return _master; }
            set { SetPropertyValue("Master", ref _master, value); }
        }
    }
}

File: VB/WinWebSolution.Module.Win/Child.vb
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Xpo
Imports System

Namespace WinWebSolution.Module.Win
    Public Class Child
        Inherits BaseObject

        Public Sub New(ByVal session As Session)
            MyBase.New(session)
        End Sub
        Private _name As String
        Public Property Name() As String
            Get
                Return _name
            End Get
            Set(ByVal value As String)
                SetPropertyValue("Name", _name, value)
            End Set
        End Property
        Private _master As Master
        <Association("Master-Children")> _
        Public Property Master() As Master
            Get
                Return _master
            End Get
            Set(ByVal value As Master)
                SetPropertyValue("Master", _master, value)
            End Set
        End Property
    End Class
End Namespace
